extensions.md

```diff
@@ -22,7 +22,7 @@ Supported keys:
 
 ```json
 "extensions": {
-    "opengdpr.mparticle.com": {
+    "OpenCompliance.mparticle.com": {
```

Comment:
Domain names are not case sensitive, so I think it makes sense to leave this as all lower case to match the rest of the domain portion.

Reply:
Agreed, will remove